prow/pod-utils/decorate/podspec_test.go

```diff
@@ -239,6 +239,7 @@ func TestProwJobToPod(t *testing.T) {
 								{Name: "BUILD_ID", Value: "blabla"},
 								{Name: "BUILD_NUMBER", Value: "blabla"},
 								{Name: "ENTRYPOINT_OPTIONS", Value: `{"args":["/bin/thing","some","args"],"timeout":7200000000000,"grace_period":10000000000,"process_log":"/logs/process-log.txt","marker_file":"/logs/marker-file.txt"}`},
+								{Name: "GOPATH", Value: "/home/prow/go"},
```

Comment:
seems slightly weird that we use `GopathEnv` versus directly "GOPATH" 🤔 /shrug

Reply:
What do you mean? This is using "GOPATH" directly.

Reply:
I guess I am of the opinion here that these vars are an API to the code under test and not coupling the test to the constant forces someone who refactors the constant to reckon with the fact that the test is broken on purpose. I don't want to test that we ingest `GoPathEnv` when decorating the `PodSpec`, I want to test that `$GOPATH` is set. Hopefully that makes sense.